package.json

```
@@ -36,7 +36,7 @@
       "!src/resources/postcss/tickets-report.pcss",
       "!src/resources/postcss/tickets-v1.pcss"
     ],
-    "jest": ["src/modules/**/__tests__/**/*.js"]
+    "jest": ["src/**/__tests__/**/*.js"]
```

You can set this to the same paths as what you have in the github workflow:

```suggestion
    "jest": [
      "src/modules/**/__tests__/**/*.js",
      "src/resources/**/__tests__/**/*.js"
    ]
```